.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build and Publish 
+
+on:
+  push:
+    branches: [ dev , master ]
+    paths-ignore: 
+      - .gradle/wrapper
+      - .gitignore
+      - CONTRIBUTING.md
+      - LICENSE
+      - THIRD PARTY NOTICES
+      - gradle.properties
+      - gradlew
+      - gradlew.bat
+      - readme.md
+      - settings.gradle
+      - Scripts/*
+      
+jobs:
+  build-and-publish:
+    env: 
+      GRADLE_OPTIONS: -x jar -x javadoc -x javadocJar -x compileJava -x processResources -x classes -x sourceJar
```

Comment:
remove all those options please as we'll need all those steps to run to get proper assets